instrumentation-client.ts

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+  dsn: 'https://36be948fdd4ed19a7814987b57b6da58@o4509152952254464.ingest.de.sentry.io/4509152954875984',
```

Comment:
Should this be on an env variable?

Reply:
I think it's okay, had the same thought but that was default from sentry codemon, I think it's simpler for now to have it directly.